src/Events/Program.cs

```diff
@@ -283,9 +283,9 @@ void ConfigureServices(IServiceCollection services, IConfiguration config)
     services.AddSingleton<ITraceLogService, TraceLogService>();
     services.AddSingleton<IAppSubscriptionService, AppSubscriptionService>();
     services.AddSingleton<IGenericSubscriptionService, GenericSubscriptionService>();
-    services.AddScoped<ICloudEventRepository, CloudEventRepository>();
-    services.AddScoped<ISubscriptionRepository, SubscriptionRepository>();
-    services.AddScoped<ITraceLogRepository, TraceLogRepository>();
+    services.AddSingleton<ICloudEventRepository, CloudEventRepository>();
```

Comment:
@hggutvik Got issues with services that are singleton cannot use repository which is scoped. 
I'm thinking rewrite of lifetime can be done in a separate issue